lib/ansible/modules/packaging/language/pip.py

```diff
@@ -95,13 +95,14 @@
     version_added: "1.3"
   executable:
     description:
-      - The explicit executable or a pathname to the executable to be used to
-        run pip for a specific version of Python installed in the system. For
+      - The explicit executable or pathname for the pip executable,
+        if different from the Ansible Python interpreter. For
         example C(pip-3.3), if there are both Python 2.7 and 3.3 installations
         in the system and you want to run pip for the Python 3.3 installation.
-        It cannot be specified together with the 'virtualenv' parameter (added in 2.1).
-        By default, it will take the appropriate version for the python interpreter
-        use by ansible, e.g. pip3 on python 3, and pip2 or pip on python 2.
+      - Mutually exclusive with I(virtualenv) (added in 2.1).
+      - Does not affect the Ansible Python interpreter.
+      - The setuptools package must be installed for the Ansible Python interpreter,
+        not for the version of Python specified by this option.
```

Comment:
This line is semi misleading.  Technically it will need to be installed for both. pip itself has a dependency on setuptools, but I would assume that if you got pip installed, setuptools should also be installed.

Reply:
ah, thanks for the clarification, I'll fix that